client/components/NewBankForm.js

```diff
@@ -60,13 +62,28 @@ export default class NewBankForm extends React.Component {
         var id = this.domId().value.trim();
         var pwd = this.domPassword().value.trim();
 
+        var customFields;
+        if (this.state.hasCustomFields) {
+            customFields = this.domCustomFields().map(function(node) {
```

Comment:
style nit: with the change below, you ought to be able to use an arrow function without {} here